packages/preset-create-react-app/src/index.ts

```diff
@@ -118,21 +119,19 @@ export const webpack = (
     },
   );
 
-  // A temporary fix to align with Storybook 6.
-  const { typescriptOptions } = options;
-  const tsDocgenRule =
-    !typescriptOptions ||
+  // NOTE: These are set by default in Storybook 6.
+  const {
+    typescriptOptions = {
```

Comment:
Can we somehow keep this in sync with Storybook defaults? If I understand this code, adding adefault for `reactDocgen` "fixes the bug", but the `reactDocgenTypescriptOptions` are still out of sync with Storybook's:

https://github.com/storybookjs/storybook/blob/next/lib/core/src/server/config/defaults.js#L4-L7

Reply:
These actually should already come through in Storybook v6 - so this is only for older Storybook versions. This default above would only occur if we didn't receive the defaults from Storybook.